integration/vite-presets-test.ts

```diff
@@ -46,16 +77,20 @@ const files = {
             }),
           },
 
+          // Ensure remixConfig is called with a frozen Remix user config
```

Comment:
It's a good thing we had a test for a frozen resolved config object. It prompted me to add this behaviour here too.